src/languages/_english.json

```diff
@@ -1263,7 +1263,10 @@
         "unsupported_network": "Notifications are only available for Ethereum Mainnet transactions at this time.",
         "change_network": "Change your Network",
         "no_first_time_permissions": "Please allow Rainbow to send you notifications.",
-        "first_time_allow_notifications": "Allow Notifications"
+        "first_time_allow_notifications": "Allow Notifications",
+        "error_alert_title": "Subscription Failed",
+        "error_alert_content": "It likely happened because of poor network conditions or because you changed settings too often.\n\nPlease try again later when your connection is more stable.",
+        "offline_alert_content": "It appears you have no internet connection.\n\nYou can't change notification settings while you're offline."
```

Comment:
Please review the copy with your native speaker skills

Reply:
"error_alert_content": "Please try again later.",
"offline_alert_content": "It appears you have no internet connection.\n\nPlease try again later.

I would go more generic/short on these